db/migrate/20230414130437_add_currency_to_invoices.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class AddCurrencyToInvoices < ActiveRecord::Migration[7.0]
+  def up
+    add_column :invoices, :currency, :string
+
+    execute <<-SQL
+      UPDATE invoices
+      SET currency = amount_currency
+    SQL
+
+    remove_column :invoices, :amount_currency
```

Comment:
Rails/BulkChangeTable: You can use `change_table :invoices, bulk: true` to combine alter queries. (https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-change_table, https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/Table.html)